spec/behavior/integration.spec.js

```diff
@@ -46,6 +46,12 @@ function rancherMock( arg, arg2 ) {
 	return promisGenerator.promise;
```

Comment:
there should be test on the `upgrade` function to test that the correct response is returned if the docker tag does not exist, and that if it does exist the `environment.getAll()` method is called